sample/src/main/java/com/davidecirillo/multichoicesample/sampleToolbar/SampleToolbarActivity.java

```diff
@@ -90,7 +90,8 @@ private void setUpAdapter(List<String> strings) {
 
         MultiChoiceToolbar multiChoiceToolbar =
                 new MultiChoiceToolbar.Builder(SampleToolbarActivity.this, toolbar)
-                        .setTitles(toolbarTitle(), "item selected")
+                        .setTitles(toolbarTitle(), "item selected") // TODO crashing when removed !?
```

Comment:
Yeah - tried that of course but then it isn't working. When only keeping the plural-setTitles() I get an exception and the list is empty (no adapter added). See comment above.